web/test/helpers/helpers-testing-library-selectors.ts

```diff
@@ -25,6 +25,32 @@ export const selectDropdownByValue = (label: string, value: string): void => {
   fireEvent.click(listbox.getByTestId(value));
 };
 
+export const selectDropdownValueByTextMouseDown = (
```

Comment:
Pebble - I have a few suggestions:
1. I think `selectLocationByText` (line 4) can be refactored away, and we use `selectDropdownByValue` instead. This can be a separate refactor.
2. The two functions created are specifically for ComboBoxes, the name should reflect this. `selectDropdownByValue` uses a mouseDown event as well, but that is specifically used for MUI Select components (non-combo boxes). The name of that function should be updated to reflect this.

Reply:
1. I'm going to leave that alone. That's outside the scope of the work. If this is a change we want to implement, let's handle that separately.
2. I can change the function names to be `selectCombobox*`, but I initially made this distinction based on previous feedback you'd provided. If you have a better name for the functions in mind, let me know and I will incorporate.